src/sentry/static/sentry/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -35,10 +35,12 @@ type Props = ReactRouter.WithRouterProps & {
   onEdit: () => void;
   renderErrorMessage?: (errorMessage: string | undefined) => React.ReactNode;
   isDragging: boolean;
+  currentWidgetDragging: boolean;
   hideToolbar?: boolean;
-  startWidgetDrag: (
-    event: React.MouseEvent<SVGElement> | React.TouchEvent<SVGElement>
-  ) => void;
+  draggableProps?: {
+    attributes: ReturnType<typeof useSortable>['attributes'];
```

Comment:
I would create something like `type UseSortableProps = ReturnType<typeof useSortable>` and reuse it here.